x-pack/plugins/rollup/__jest__/integration_tests/job_create.test.js

```diff
@@ -0,0 +1,1298 @@
+        });
+      });
+
+      describe('rollup index name', () => {
+        beforeEach(() => {
+          expect(findTestSubject('rollupJobNextButton').props().disabled).toBe(false);
+        });
+
+        afterEach(() => {
+          expect(findTestSubject('rollupJobNextButton').props().disabled).toBe(true);
+        });
+
+        it('should not allow spaces', () => {
+          form.setInputValue('rollupIndexName', 'with space');
+          userActions.clickNextStep();
+          expect(getFormErrorsMessages()).toContain('Remove the spaces from your rollup index name.');
+        });
+
+        it('should not allow invalid characters', () => {
+          const expectInvalidChar = (char) => {
+            form.setInputValue('rollupIndexName', `rollup_index_${char}`);
+            userActions.clickNextStep();
+            expect(getFormErrorsMessages()).toContain(`Remove the characters ${char} from your rollup index name.`);
+          };
+
+          [...INDEX_PATTERN_ILLEGAL_CHARACTERS_VISIBLE, ','].reduce((promise, char) => {
+            return promise.then(() => expectInvalidChar(char));
+          }, Promise.resolve());
+        });
+
+        it('should not allow a dot as first character', () => {
+          form.setInputValue('rollupIndexName', '.kibana');
+          userActions.clickNextStep();
+          expect(getFormErrorsMessages()).toContain('Index names cannot begin with periods.');
+        });
+      });
+
+      describe('rollup cron', () => {
+        const changeFrequency = (value) => {
+          findTestSubject('rollupJobCreateFrequencySelect').simulate('change', { target: { value } });
+        };
+
+        const generateStringSequenceOfNumbers = (total) => (
+          new Array(total).fill('').map((_, i) => i < 10 ? `0${i}` : i.toString())
+        );
+
+        describe('frequency', () => {
+          it('should allow "minute", "hour", "day", "week", "month", "year"', () => {
+            const frequencySelect = findTestSubject('rollupJobCreateFrequencySelect');
+            const options = frequencySelect.find('option').map(option => option.text());
+            expect(options).toEqual(['minute', 'hour', 'day', 'week', 'month', 'year']);
+          });
+
+          describe('every minute', () => {
+            it('should not have any additional configuration', () => {
+              changeFrequency(MINUTE);
+              expect(findTestSubject('rollupCronFrequencyConfiguration').length).toBe(0);
+            });
+          });
+
+          describe('hourly', () => {
+            beforeEach(() => {
+              changeFrequency(HOUR);
+            });
+
+            it('should have 1 additional configuration', () => {
+              expect(findTestSubject('rollupCronFrequencyConfiguration').length).toBe(1);
+              expect(testSubjectExists('rollupJobCreateFrequencyHourlyMinuteSelect')).toBe(true);
+            });
+
+            it('should allow to select any minute from 00 -> 59', () => {
```

Comment:
I've seen Jest tests tend to use `test` over `it`. It might help people differentiate between a Mocha and Jest test at a glance if we use `test` here.

Reply:
I prefer to stick to the [RSpec](https://en.wikipedia.org/wiki/RSpec) and use `describe` and `it`. (see comment here https://stackoverflow.com/a/54907283/5642633). Aside, if I'm not wrong, files ended in `.test.js` are all Jest, no?

Reply:
Yeah, Jest test files end in `.test.js`. But in terms of optimizing for readability, it reduces a bit of friction when it's easier to identify code by just looking at it than by looking at the file name. And if you look at any of the [test docs](https://jestjs.io/docs/en/getting-started), they all use `test`. Not a blocker though, just pointing it out. 😄

Reply:
> in terms of optimizing for readability

Funny, when I _read_ a test (`to.equal` vs `toEqual`) I don't wonder what is the testing framework underneath. I do when I _write_ it obviously.  😄